opm/material/common/UniformXTabulated2DFunction.hpp

```diff
@@ -417,14 +417,14 @@ class UniformXTabulated2DFunction
     {
         if (xPos_.empty() || xPos_.back() < nextX) {
             xPos_.push_back(nextX);
-            yPos_.push_back(-1e100);
+            yPos_.push_back(-1e30);
```

Comment:
Might it make sense to use `numeric_limits<Scalar>::lowest() / 2` or something to that effect here?  Maybe even `numeric_limits<Scalar>::quiet_NaN()`?  For what it's worth, the constant `-1.0e30` is type `double` regardless of what `Scalar` is.

Reply:
i can do that. i am aware there's a type conversion but the other way would warn. i'm not exactly sure how the sentinel is used so i'm reluctant to use quiet_NaN.

Reply:
done.